sycl/source/device_selector.cpp

```diff
@@ -77,6 +77,11 @@ device device_selector::select_device() const {
                                 PI_DEVICE_NOT_FOUND);
 }
 
+/// Devices of different kinds are prioritized in the following order:
+/// 1. GPU
+/// 2. Accelerator
```

Comment:
That's no longer the case for accelerators as of https://github.com/intel/llvm/commit/b217bc5b22064d074dc75a01657593323efc9c71

Reply:
@sergey-semenov is it ok if I fix this right after this PR lands?

Reply:
Sure